examples/SimpleViewer/RiveInspectorView.qml

```diff
@@ -71,13 +71,6 @@ Item {
             Layout.minimumHeight: 200
         }
 
-        Item {
```

Comment:
Well the reason to have this was to ensure that there is a reasonable Space between the elements, so dont remove it

Reply:
I will make some space around the rive item, but this item on just one side just made it look broken. I would suggest to add some padding and different shade of gray as the background, so it is clear, where the borders are. (or just have borders around the item to make it even more apparent)